services/parser.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+class Parser:
+    def get_document(self, path: str):
+        _, extension = os.path.splitext(path)
+        match extension[1:]:
+            case "txt":
+                with open(path, "r") as file:
+                    doc = file.read()
+            case "pdf":
+                pass
+            case "doc":
+                pass
+            case "docx":
+                pass
+            case "xml":
+                pass
+        return doc
+
+    def get_document_chunks(self, doc: str, chunk_size: int):
+        size = chunk_size
+        chunks = []
+        current_chunk = []
+        text = doc.replace("\n", " ")
+        j = 0
+
+        for i in range(len(text)):
```

Comment:
Should add comments here or a pydoc comment explaining what this function does

Reply:
I went ahead and added some pydoc comments and documented the class and the function to explain what it does.